2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_setup.py:_flush():67] Configure stats pid to 136867
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/.config/wandb/settings
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/Documents/GitHub/Bachelor/wandb/settings
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250301_195329-5y6dqfe7/logs/debug.log
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250301_195329-5y6dqfe7/logs/debug-internal.log
2025-03-01 19:53:29,789 INFO    MainThread:136867 [wandb_init.py:init():761] calling init triggers
2025-03-01 19:53:29,790 INFO    MainThread:136867 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'model': 'UNet', 'dataset': 'cropped_objects', 'batch_size': 6, 'val_batch_size': 2, 'learning_rate': 0.0001, 'global_image_size': (650, 650), 'num_epochs': 10, 'gaussian_blur': True, '_wandb': {}}
2025-03-01 19:53:29,790 INFO    MainThread:136867 [wandb_init.py:init():784] starting backend
2025-03-01 19:53:29,790 INFO    MainThread:136867 [wandb_init.py:init():788] sending inform_init request
2025-03-01 19:53:29,795 INFO    MainThread:136867 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-01 19:53:29,795 INFO    MainThread:136867 [wandb_init.py:init():803] backend started and connected
2025-03-01 19:53:29,797 INFO    MainThread:136867 [wandb_init.py:init():896] updated telemetry
2025-03-01 19:53:29,803 INFO    MainThread:136867 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-01 19:53:30,246 ERROR   MainThread:136867 [wandb_init.py:init():1481] error in wandb.init()
Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1471, in init
    return wi.init(run_settings, run_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 968, in init
    raise error
wandb.errors.errors.CommError: failed to upsert bucket: returned error 403 Forbidden: {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-03-01 19:53:30,257 WARNING MsgRouterThr:136867 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-01 19:53:30,258 INFO    MsgRouterThr:136867 [mailbox.py:close():115] Closing mailbox, abandoning 0 handles.
